### PR TITLE
Toggl Track: Sort time entries starting from latest

### PR DESCRIPTION
**Pull request by rogatty**
## Description

Time entries were in a random order, and now they're sorted starting from the latest one. It makes it easier to continue work after having a break.

Documentation for the API field I have used: https://github.com/toggl/toggl_api_docs/blob/54a59433c3eb79a38bd69df2be30b67e21a7b232/chapters/time_entries.md?plain=1#L18

Documentation for the `sort()` function: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by bkeys818 — APPROVED** [review]
Looks good to me

**Comment by pernielsentikaer** [comment]
Hi @rogatty 👋 

Thanks for the fix 🔥 

Do you want to check it out @franzwilhelm 😊

File: extensions/toggl-track/src/index.tsx [hunk 1]
```diff
@@ -15,13 +15,25 @@ function ListView() {
...
-  }, [] as TimeEntry[]);
+  const timeEntriesWithUniqueProjectAndDescription = timeEntries
+    .reduce((acc, timeEntry) => {
+      const existing = acc.find((t) => t.description === timeEntry.description && t.pid === timeEntry.pid);
+      if (!existing) {
+        acc.push(timeEntry);
+      }
+      return acc;
+    }, [] as TimeEntry[])
+    .sort((a, b) => {
```
Comment (rogatty) [inline extensions/toggl-track/src/index.tsx:26]: Actually, this is wrong. It should be sorted first and then filtered. I'll update it later today.

Comment (rogatty) [reply extensions/toggl-track/src/index.tsx]: [`4efb10c` (#4450)](https://github.com/raycast/extensions/pull/4450/commits/4efb10c0cb5159318f7bfd9b56a6627ba5ded1e0)

**Review by franzwilhelm — CHANGES_REQUESTED** [review]
Nice catch! Just some cleanup suggestions here :)

**Comment by rogatty** [comment]
@franzwilhelm thanks for the code review, I cleaned it up.

**Review by franzwilhelm — APPROVED** [review]
🎉 🙌

**Comment by franzwilhelm** [comment]
@pernielsentikaer ready to merge

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**